exmg-paper-combobox.ts

```diff
@@ -247,17 +294,22 @@ export class PaperComboboxElement extends LitElement {
     this.token = {id, text};
   }
 
-  filterItems() {
+  filterItems(): boolean {
     const items: NodeListOf<HTMLElement> = this.querySelectorAll('paper-item, paper-icon-item');
     const hasFilterPhrase = !!this.inputValue && this.inputValue.length > 0;
+    const phrase = hasFilterPhrase ? this.inputValue.toLowerCase().trim() : '';
+    let isAnyItemActive = false;
 
     items.forEach(item => {
-      if (hasFilterPhrase && item.textContent &&  item.textContent.indexOf(this.inputValue || '') === -1) {
+      if (hasFilterPhrase && item.textContent &&  item.textContent.toLowerCase().indexOf(phrase) === -1) {
```

Comment:
I did not analyze it deeply, but you use hasFilterPhrase that is outside item's scope inside foreach? Are you sure there is everything alright with logic? maybe there is need to move it to 1 if outside foreach?

Reply:
It is on purpose, don't need to repeat check in loop `!!this.inputValue && this.inputValue.length > 0` i can determine it before looping. Then there is more readable and a bit but more performant